ci/.gitignore

```diff
@@ -0,0 +1,2 @@
+credentials.yaml
```

Comment:
i don't see this line in https://github.com/siredmar/bb-buildingblock/blob/ci/.gitignore and locally?!

Reply:
solved ... it's another file